### DISCUSSION TITLE
How ti get a multichoice list set of ckecked items

### DISCUSSION THREAD

**Discussion by ocsike** [eltos/SimpleDialogFragments#95] [Q&A] [answered]
Hi,
I've created a CustomListDialog class and works fine. I need to get checkedids/checkeditems in onPositiveButtonClick method, but it's empty.  I tried getListView().getCheckedItemIds() and no luck.   What is wrong in my class?
Thanks

**Comment by eltos** [comment, marked as answer]
Use `mAdapter.getCheckedItemIds();`

**Reply by ocsike** [reply]
Many thanks 👍
